src/PrestaShopBundle/Resources/config/routing/admin/sell/catalog/categories.yml

```diff
@@ -1,10 +1,11 @@
 admin_categories_index:
-    path: /
+    path: /{categoryId}
     methods: [GET, POST]
     defaults:
         _controller: PrestaShopBundle:Admin\Sell\Catalog\Category:index
         _legacy_controller: AdminCategories
         _legacy_link: AdminCategories
+        categoryId: ''
```

Comment:
Feels weird having id as empty string. :confused:

Reply:
this is for the initial category, in order to keep the same url format

Reply:
inject null instead

Reply:
Alright done, and I added requirements on `categoryId` because it cause conflicts with search url matching causing a bug when using filters